cmd/storj-sim/network.go

```diff
@@ -310,7 +311,7 @@ func newNetwork(flags *Flags) (*Processes, error) {
 			)
 		}
 		for flag, db := range redisDBs {
```

Comment:
note, you can inline these

Reply:
ok, in that case, should i just change the redisDBs map to just be a `const`?

Reply:
To the arguments starting at lines 282, you can add:

```
"--live-accounting.storage-backend": "redis://" + redisServers[i] + "?db=0",
"--server.revocation-dburl":         "redis://" + redisServers[i] + "?db=1",
```

Reply:
perfect